app/actions/remote/groups.ts

```diff
@@ -4,13 +4,13 @@
 import {Client} from '@client/rest';
 import NetworkManager from '@managers/network_manager';
 
-export const getGroupsForAutocomplete = async (serverUrl: string, channelId: string) => {
+export const getGroupsForAutocomplete = async (serverUrl: string, query: string) => {
     let client: Client;
     try {
         client = NetworkManager.getClient(serverUrl);
     } catch (error) {
         return [];
     }
 
-    return client.getAllGroupsAssociatedToChannel(channelId, true);
+    return client.getGroups(query);
```

Comment:
I assumed its because we need to get all groups, so I'm not sure how we are to handle the team / channel constraints alongside custom groups. Will circle back,